### DISCUSSION TITLE
Anti-fingerprinting techniques

### DISCUSSION THREAD

**Discussion by seancsy** [fingerprintjs/fingerprintjs#848] [General]
I am researching ip+fingerprint detection websites, and currently see  whoer, ipdog.io and browserleak, which are similar to each other. I don’t know which website’s detection results are more valuable?

**Comment by makma** [comment]
Hey @seancsy, I don't really understand your question. Please share what specific scenario you want to achieve and links to the services you consider.

**Reply by seancsy** [reply]
For example, I want to do some things on social media, but I don't want to be tracked, that's all, so, a friend recommended this website to me, but I don't know how to use it yet~
[https://ipdog.io/](url)

**Reply by makma** [reply]
This project/repository allows anyone to research how browser fingerprinting works, not how to avoid it.
Nevertheless, a good starting point for your research might be this article: https://fingerprint.com/blog/browser-anti-fingerprinting-techniques/
